tests/unit_tests/storage/test_local_storage.py

```diff
@@ -128,37 +127,6 @@ def test_refresh(tmp_path):
             assert _cases(accessor) == _cases(reader)
 
 
-def test_runtime_types(tmp_path):
-    with open_storage(tmp_path) as storage:
-        assert isinstance(storage, local.LocalStorageReader)
-        assert not isinstance(storage, local.LocalStorageAccessor)
-
-    with open_storage(tmp_path, mode="r") as storage:
-        assert isinstance(storage, local.LocalStorageReader)
-        assert not isinstance(storage, local.LocalStorageAccessor)
-
-    with open_storage(tmp_path, mode="w") as storage:
-        assert isinstance(storage, local.LocalStorageReader)
-        assert isinstance(storage, local.LocalStorageAccessor)
```

Comment:
Why are these removed?

Reply:
You get the same type regardless of mode, so the test makes no sense anymore.

Reply:
But is there any tests of the input mode now? The test would have to be rewritten, could be something like:
```python
assert storage.mode == expected
```